src/components/report/CharGrid.vue

```diff
@@ -31,7 +40,40 @@
 				@updateVariableStyles="updateVariableStyles"
 				class="char-sliders"
 			/>
-			<ol class="grid" :style="variableStyles">
+
+			<template v-if="showCategories">
+				<div v-for="(cat, index) in catChars" :key="`cat${index}`">
+					<h3 class="title">
+						{{ cat.category }}
+						<template v-if="cat.subCategory || cat.script">
+							—
+							{{ cat.subCategory }}
+							{{ cat.script | capitalize }}
+						</template>
+					</h3>
+
+					<ol class="grid" :style="variableStyles">
+						<!-- Put this back → :class="char.feature ? 'feature' : 'code'" -->
+						<li
+							v-for="char in cat.chars"
+							:key="`${char}`"
+							class="code"
+						>
+							<span class="char" v-html="entitify(char)"></span>
+							<span class="label" v-if="!char.feature">{{
+								char
+							}}</span>
+							<!--
+						<span class="label" v-if="char.feature">{{
+							char.feature
+						}}</span>
+						-->
```

Comment:
Is this intentionally commented out? Can it be removed?

Reply:
This is commented out until we either implement this, or delete this functionality. It will show variants of a character (e.g. stylistic sets) when enabled. This will happen outside this particular PR.